powersimdata/input/exporter/export_to_pypsa.py

```diff
@@ -7,6 +7,16 @@
 from powersimdata.utility.helpers import _check_import
 
 
+def restore_original_columns(df, overwrite=[]):
```

Comment:
This is minor: we might what to avoid immutable defaults for kwargs, i.e. `overwrite=None` with the logic in the function `if not overwrite: overwrite = []`.

Reply:
It will be taken care of after it is merged